metaflow/plugins/kfp/tests/flows/s3_sensor_flow.py

```diff
@@ -0,0 +1,43 @@
+from metaflow import FlowSpec, step, resources, s3_sensor, Parameter
+
+
+def formatter(path: str, flow_parameters_json: dict) -> str:
+    import os
+
+    path = path.replace("CODE_PACKAGE_ID", flow_parameters_json["code_package_id"])
+    path = path.replace("POD_NAMESPACE", os.environ["POD_NAMESPACE"])
```

Comment:
why use replace when you could use python format? see https://realpython.com/python-formatted-output/